root/admin/wikidoc/DeleteWikiDoc.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *     | o  ((*   l  /
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import FormSubmit from '../../components/FormSubmit';
+import Layout from '../../layout';
+import {withCatalystContext} from '../../context';
+
+type Props = {
+  +$c: CatalystContextT,
+  +form: FormT<{}>,
+  +page: string,
+};
+
+const DeleteWikiDoc = withCatalystContext(({
+  $c,
+  form,
+  page,
+}: Props) => (
+  <Layout fullWidth title={l('Remove Page')}>
+    <div id="content">
+      <h1>{l('Remove Page')}</h1>
+      <p>
+        {exp.l(`Are you sure you wish to remove the page
+                “{page_uri|{page_name}}” from the transclusion table?`,
+               {
+                 page_name: page,
+                 page_uri: '/doc/' + encodeURIComponent(page),
+               })}
+      </p>
+      <form action={$c.req.uri} method="post">
+        <FormSubmit label={l('Yes, I\'m sure')} name="confirm.submit" />
```

Comment:
`FormSubmit` might need `value="1"` added to the button. I needed it added in https://github.com/metabrainz/musicbrainz-server/commit/5215f24fe7f0ebe9442d8a29b7b861121362ce03 to fix the WikiDoc delete test.

Reply:
Readded in https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/f1dd31a6b4cbf10cf24cbca7c75c2c6a491dc2d9

Reply:
Also required to add https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/96c7171082254da72c54b4504363b5c5c809f196